### PR TITLE
[inference][trt]modify test timeout  and test_trt_convert_activation bug fix

### PR DESCRIPTION
**Pull request by bukejiyu**
### PR types
Others

### PR changes
Others

### Description
Pcard-71501
timeout modify:
test_trt_conv_quant_dequant_pass(test:379)、
test_trt_matmul_quant_dequant(test 433.670s) 
test_simplify_with_basic_ops_pass_autoscan(test:226.878)、
test_trt_dynamic_shape_ernie(test:451.04)、
test_trt_convert_activation and test_fused_multi_transformer_int8_op bugfix
comment out int8 mode test_trt_dynamic_shape_groupnorm

### CODE REVIEW

**Review by zhangjun — DISMISSED** [review]
LGTM

**Review by XieYunshen — APPROVED** [review]
LGTM

**Review by XieYunshen — DISMISSED** [review]
LGTM

File: test/ir/inference/CMakeLists.txt [hunk 1]
```diff
@@ -197,8 +197,8 @@ if(WITH_GPU AND TENSORRT_FOUND)
   set_tests_properties(test_trt_tile_op PROPERTIES TIMEOUT 60)
   set_tests_properties(test_trt_fc_fuse_quant_dequant_pass PROPERTIES TIMEOUT
                                                                       100)
-  set_tests_properties(test_trt_conv_quant_dequant_pass PROPERTIES TIMEOUT 180)
-  set_tests_properties(test_trt_matmul_quant_dequant PROPERTIES TIMEOUT 450)
+  set_tests_properties(test_trt_conv_quant_dequant_pass PROPERTIES TIMEOUT 100)
+  set_tests_properties(test_trt_matmul_quant_dequant PROPERTIES TIMEOUT 180)
```
Comment (zhangjun) [inline test/ir/inference/CMakeLists.txt]: 为什么又改回去了？

File: test/ir/inference/CMakeLists.txt [hunk 2]
```diff
@@ -219,7 +219,7 @@ if(WITH_GPU AND TENSORRT_FOUND)
     set_tests_properties(test_transfer_layout_elim_pass PROPERTIES TIMEOUT 300)
 
     set_tests_properties(test_simplify_with_basic_ops_pass_autoscan
-                         PROPERTIES TIMEOUT 60)
+                         PROPERTIES TIMEOUT 240)
```
Comment (zhangjun) [inline test/ir/inference/CMakeLists.txt:222]: 同上

**Review by zhangjun — APPROVED** [review]

**Pull request merged**